tests/ui/parser/trailing-carriage-return-in-string.stderr

```diff
@@ -1,7 +1,7 @@
 error: unknown character escape: `\r`
   --> $DIR/trailing-carriage-return-in-string.rs:10:25
    |
-LL |     let bad = "This is \
 a test";
+LL |     let bad = "This is \␍ a test";
    |                         ^ unknown character escape
```

Comment:
This is confusing, because the problem is that `\` escape is followed by a non-printable character. This is parsed as `\␍`, which is kind-of `\\r`.